Makefile

```diff
@@ -1,16 +1,19 @@
-lint: bandit flake8 isort
-	
+lint: bandit black flake8 isort
+
 bandit:
 	pipenv run bandit -r asaps
+
+black:
+	pipenv run black --check --diff asaps tests
```

Comment:
For this repo we can run these (except bandit) on the whole directory. It's only alma-scripts where that's an issue because there are other folders that people are using for other things that we should ignore for now.